src/main/java/hudson/plugins/ansicolor/AnsiHtmlOutputStream.java

```diff
@@ -473,6 +476,7 @@ private String getRgbColor(int r, int g, int b) throws IOException {
         return "#" + String.format("%02X", r) + String.format("%02X", g) + String.format("%02X", b);
     }
 
+    @SuppressFBWarnings("UC_USELESS_CONDITION")
```

Comment:
I think that the useless condition is `if (paletteIndex < 256)`.
The SuppressFBWarnings annotation will suppress all useless condition warnings in this function, isn't it?
IMHO it should only suppress the single location of `if (paletteIndex < 256)`.